replayer/tests/functional_tests/binary_scripts/read.c

```diff
@@ -0,0 +1,13 @@
+#include <signal.h>
+#include <stdio.h>
+#include <unistd.h>
+
+
+int main() {
+  char buffer[100];
+  printf("%lu\n", (unsigned long)buffer);
+  fflush(stdout);
+  kill(getpid(), SIGSTOP);
```

Comment:
Why not raise again?

Reply:
If you do raise, after SIGCONT is sent, the process calls rt_sigprocmask

Reply:
(This is a test script btw, you need to understand it in context of the tests)